src/mrack/providers/provider.py

```diff
@@ -224,9 +232,17 @@ async def parse_error_hosts(self, hosts):
                 f"{self.dsp_name}: Host - {host.host_id}\tStatus - {host.status}"
             )
 
+            if host.status != STATUS_ACTIVE:
+                errors.append(host)
+                continue
+
             # self._wait_for_ssh checks the host's capability to be
             # connected to using ssh; if the check fails we reprovision
-            if host.status != STATUS_ACTIVE or not await self._wait_for_ssh(host):
+            if not await self._wait_for_ssh(host):
```

Comment:
If host is not active we should not check its connectivity, right?

Reply:
I thought this continue helps https://github.com/neoave/mrack/pull/99/files#diff-9eb08e19e12ca58d0f3b3a61d2426fbd72ebefbd5e7171981e5fa4cbac237e78R237

Reply:
You are right, I overlooked the for loop.